tensorflow/core/platform/windows/env.cc

```diff
@@ -52,7 +52,16 @@ class StdThread : public Thread {
 
 class WindowsEnv : public Env {
  public:
-  WindowsEnv() {}
+  WindowsEnv()
+      : GetSystemTimePreciseAsFileTime_(NULL) {
+    HMODULE module = GetModuleHandle("kernel32.dll");
```

Comment:
We do it this way because GetSystemTimePreciseAsFileTime() function is only available in Windows 8/Server 2012 and later. We still want the Windows port to work for earlier versions though.